styles/06-components/modal.css

```diff
@@ -0,0 +1,61 @@
+/* .modal {
+  height:  100%;
+  width: 100%;
+  position: fixed;
+  z-index: 1;
+  top: 30;
+  border: 1px solid black;
+  overflow: auto;
+}
+
+.modal .list-stacked{
+  margin: auto;
+  border: 0;
+  /* width: 70%; */
+  /* box-shadow: 0 5px 8px 0 var(--color-shadow), 0 7px 20px 0 #0000002b; */
+/* }
+
+.modal {
+  width: clamp(10rem, 20rem, 30rem);;
+  
+}
+
+.modal .icon {
+  color: var(--color-secondary)
+} */
```

Comment:
Remove the comments if it is of no use.

Reply:
I'll need it to make it responsive later